py/desisim/scripts/quickquasars.py

```diff
@@ -79,6 +79,7 @@ def parse(options=None):
     parser.add_argument('--balprob',type=float,required=False, help="To add BAL features with the specified probability (e.g --balprob 0.5). Expect a number between 0 and 1 ")
     parser.add_argument('--no-simqso',action = "store_true", help="Does not use desisim.templates.SIMQSO to generate templates, and uses desisim.templates.QSO instead.")
     parser.add_argument('--no-transmission',action = 'store_true', help='Do not multiply continuum by transmission, use F=1 everywhere')
+    parser.add_argument('--modified-emline',action = 'store_true', help='Do not multiply continuum by transmission, use F=1 everywhere')
```

Comment:
@alxogm, the comment does not correspond. But anyway, I don't think we need an option. Let's use the new model always, there are no obvious reasons to use sometime the old one. Is there?

Reply:
Yes, I'll fix it (remove it)